graylog2-server/src/main/java/org/graylog2/shared/journal/LocalKafkaJournal.java

```diff
@@ -795,6 +796,14 @@ public long getNextReadOffset() {
         return nextReadOffset;
     }
 
+    public double getJournalUtilization() {
```

Comment:
Is there a reason not to pull this method into the `Journal` interface? It seems fairly generic so I think it would make sense. Then we can keep using the interface when we want to call this method.

Reply:
I will pull it up